geps/overview.md

```diff
@@ -8,8 +8,10 @@ process for the main Kubernetes project:
 1. Make changes and proposals discoverable (current and future).
 1. Document design ideas, tradeoffs, decisions that were made for
   historical reference.
+1. Record the results of larger community discussions.
+1. Record changes to the GEP process itself.
 
-## Process
+## Process diagram
```

Comment:
Tiny nit, I think I prefer the single word title because it's easier to link to + preserves any existing links that may already target this section.